vmdb/product/toolbars/ems_infra_center_tb.yaml

```diff
@@ -22,6 +22,11 @@
       :text: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :title: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :url: '/edit'
+    - :button: ems_infra_scale
```

Comment:
@rwsu please make sure the button is visible only for OpenstackInfra provider by adding the condition ^

@martinpovolny this will need to be re-designed as part of plug-ability I think